src/helpers/data.js

```diff
@@ -58,12 +45,13 @@ export default {
     if (!props.data) {
       return [];
     }
-    const accessor = Helpers.createAccessor(has(props, axis) ? props[axis] : axis);
-    const dataStrings = (props.data)
-        .map((datum) => accessor(datum))
-        .filter((datum) => typeof datum === "string");
-    // return a unique set of strings
-    return compact(uniq(dataStrings));
```

Comment:
uniq() is the kind of method that from my experience it's good to keep, it is significantly faster than `arr.reduce( if (indexOf) push )`